code/examples/main-can_bus_demo.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+
+This main is used to test the module-scheduler
+For this programm to compile, the supplied Makefile must be used
+
+This program makes the can-bus demo run on one arduino using the
+module-scheduler
+
+for the button, I/O pin 53 should be used, there is no external pullup resistor
+needed. by pushing the button, the built-in led should turn on, by releasing the
+button, the led should turn off.
+
+*/
+
+#include <hwlib.hpp>
+#include <rtos.hpp>
+
+#include <can_bus_pending_task.hpp>
+#include <module_task.hpp>
+
+#include <button/module.hpp>
+#include <controller/module.hpp>
+#include <led/module.hpp>
+
+namespace r2d2::module_scheduler {
```

Comment:
You might want to add more comments, people who are going to view this example aren't going to know what is going on all that well.

Reply:
https://github.com/R2D2-2019/module_scheduler/pull/2/commits/439cf0d0d60e02638883046be3bac7eb6e19c228